From d7c7312011dd1e2a2e7ea3b51c05a56fd0ce0699 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Fri, 17 Jun 2016 01:00:46 +0000
Subject: [PATCH 2/2] Fix build with gcc-6

cpputest/3.5+gitAUTOINC+ee14dc0646-r3/git/src/CppUTestExt/MockSupport.cpp:277:5: error: this 'if' clause does not guard... [-Werror=misleading-indentation]
     if(lastActualFunctionCall_)
     ^~
---
 src/CppUTestExt/MockSupport.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CppUTestExt/MockSupport.cpp b/src/CppUTestExt/MockSupport.cpp
index 75fbdde..d41ccea 100644
--- a/src/CppUTestExt/MockSupport.cpp
+++ b/src/CppUTestExt/MockSupport.cpp
@@ -274,8 +274,8 @@ void MockSupport::failTest(MockFailure& failure)
 
 void MockSupport::checkExpectationsOfLastCall()
 {
-    if(lastActualFunctionCall_)
-        lastActualFunctionCall_->checkExpectations();
+	if(lastActualFunctionCall_)
+		lastActualFunctionCall_->checkExpectations();
 
 	for(MockNamedValueListNode *p = data_.begin();p;p = p->next())
 		if(getMockSupport(p) && getMockSupport(p)->lastActualFunctionCall_)
-- 
1.9.1

